packages/react-native-editor/bin/build_e2e_ios_app

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -eu
+
+DEFAULT_DESTINATION='platform=iOS Simulator,name=iPhone 13'
+if [[ -z "${RN_EDITOR_E2E_IOS_DESTINATION-}" ]]; then
+	DESTINATION="$DEFAULT_DESTINATION"
+else
+	DESTINATION="$RN_EDITOR_E2E_IOS_DESTINATION"
+fi
+
+# Define xcodebuild in a variable so that we can conditionally pipe it through
+# the xcbeautify output parser
+XCODEBUILD_CMD="SKIP_BUNDLING=true \
+	xcodebuild \
+	-workspace ios/GutenbergDemo.xcworkspace \
+	-configuration Release \
+	-scheme GutenbergDemo \
+	-derivedDataPath ios/build/GutenbergDemo \
+	-destination '$DESTINATION'"
+
+# Check if xcbeautify is available
+if command -v xcbeautify >/dev/null 2>&1; then
+	# Pipe the output to xcbeautify
+	eval "$XCODEBUILD_CMD" | xcbeautify
+else
+	# Run xcodebuild without piping the output
+	eval "$XCODEBUILD_CMD"
+fi
```

Comment:
I extracted the `xcodebuild` call for two reasons:

1. Allow overriding the `-destination` parameter
2. Use `xcbeautify` to parse the output to make local debugging easier